src/software/geom/algorithms/generate_velocity_obstacle.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <vector>
+
+#include "software/ai/navigator/path_planner/hrvo/velocity_obstacle.h"
+#include "software/geom/circle.h"
+#include "software/geom/polygon.h"
+
+/**
+ * Set of functions that generate velocity obstacles for an obstacle
+ *
+ * @param obstacle Static obstacle
```

Comment:
are these methods specifically for static obstacles?

Reply:
Good point. Given that you also pass in the obstacle velocity, it can support static or dynamic obstacles. I've updated the comment to be more generalized